### DISCUSSION TITLE
Urgent: 500 Internal Server Error and Missing Previous Projects

### DISCUSSION THREAD

**Discussion by sachinbhartiaphro** [codemagic-ci-cd/codemagic-docs#2543] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Dear  Support Team,

I hope this message finds you well. I am writing to seek urgent assistance regarding an issue I encountered with your platform.

1. **500 Internal Server Error:** Upon creating a new project today, I encountered a 500 Internal Server Error. This issue persists and prevents me from progressing with the project setup.
2. **Missing Previous Projects:** Additionally, I have noticed that all my previously created projects are no longer visible in my account. These projects were accessible and functioning properly as recently as yesterday.
Given that both issues are significantly impacting my workflow, I kindly request your immediate attention and resolution.

**Details:**

Username/Email: sachin@aphroecs.com
Date of Issue: 20 May 2024
Browser/OS: google chrome

**Steps to Reproduce the Error:**
Log in to my Code Magic account.
Attempt to create a new project.
Observe the 500 Internal Server Error message.
I would appreciate any guidance or support you can provide to resolve these issues promptly. If you require any further information or need me to perform any specific actions, please let me know.

Thank you for your prompt attention to this matter.

Best regards,

SachinBharti
email: sachin@aphroecs.com

**Comment by dtrdic** [comment]
Hey @sachinbhartiaphro, there was an issue with BitBucket integration and it has been resolved now. Try to log in again, should be all right.

**Comment by sachinbhartiaphro** [comment]
Hey @dtrdic 500 error issues were resolved but we created a new project in code magic . but projects not showing on my dashboard.

**Reply by dtrdic** [reply]
hey @sachinbhartiaphro please open a different thread since this is a different issue. I'll close this one as the original issue has been resolved
